### ISSUE TITLE
Mass Obsoletion: focus on branch 'cancer or benign tumor' (MONDO:0045024)

### ISSUE THREAD

**Issue by sabrinatoro** [monarch-initiative/mondo#6739]
Branch: 'cancer or benign tumor' (MONDO:0045024)

Terms to obsolete: 

obsoletion candidate ID | obsoletion candidate label
-- | --
http://purl.obolibrary.org/obo/MONDO_0018532 | adenocarcinoma of liver and intrahepatic biliary tract
http://purl.obolibrary.org/obo/MONDO_0015818 | aggressive primary cutaneous B-cell lymphoma
http://purl.obolibrary.org/obo/MONDO_0015817 | aggressive primary cutaneous T-cell lymphoma
http://purl.obolibrary.org/obo/MONDO_0020205 | bulbar conjunctival dermoid or conjunctival dermolipoma
http://purl.obolibrary.org/obo/MONDO_0018720 | common cystic lymphatic malformation
http://purl.obolibrary.org/obo/MONDO_0016708 | embryonal tumor of neuroepithelial tissue
http://purl.obolibrary.org/obo/MONDO_0028618 | gastroenteric neuroendocrine neoplasm
http://purl.obolibrary.org/obo/MONDO_0018188 | hereditary intestinal polyposis
http://purl.obolibrary.org/obo/MONDO_0017129 | inherited cardiac tumor
http://purl.obolibrary.org/obo/MONDO_0018538 | inherited digestive cancer-predisposing syndrome
http://purl.obolibrary.org/obo/MONDO_0017128 | inherited digestive tract tumor
http://purl.obolibrary.org/obo/MONDO_0035645 | inherited gynecological cancer-predisposing syndrome
http://purl.obolibrary.org/obo/MONDO_0025511 | inherited neuroendocrine tumor
http://purl.obolibrary.org/obo/MONDO_0017891 | inherited renal cancer-predisposing syndrome
http://purl.obolibrary.org/obo/MONDO_0015950 | inherited skin tumor
http://purl.obolibrary.org/obo/MONDO_0017127 | inherited soft tissue tumor
http://purl.obolibrary.org/obo/MONDO_0017036 | Langerhans cell histiocytosis in childhood and adulthood
http://purl.obolibrary.org/obo/MONDO_0034872 | large granular lymphocyte leukemia
http://purl.obolibrary.org/obo/MONDO_0020084 | lymphoproliferative disease associated with primary immune disease
http://purl.obolibrary.org/obo/MONDO_0020081 | macrophage or histiocytic tumor
http://purl.obolibrary.org/obo/MONDO_0020181 | mesenchymatous palpebral tumor
http://purl.obolibrary.org/obo/MONDO_0016726 | neuronal tumor
http://purl.obolibrary.org/obo/MONDO_0016701 | oligoastrocytic tumor
http://purl.obolibrary.org/obo/MONDO_0020178 | palpebral lentiginosis
http://purl.obolibrary.org/obo/MONDO_0020180 | palpebral piliary tumor
http://purl.obolibrary.org/obo/MONDO_0020176 | palpebral sebaceous gland tumor
http://purl.obolibrary.org/obo/MONDO_0020177 | pigmented palpebral tumor
http://purl.obolibrary.org/obo/MONDO_0016721 | pineal tumor of neuroepithelial tissue
http://purl.obolibrary.org/obo/MONDO_0020174 | precancerous lesion of palpebral epidermis
http://purl.obolibrary.org/obo/MONDO_0016738 | primary germ cell tumor of central nervous system
http://purl.obolibrary.org/obo/MONDO_0015683 | primary malignant peritoneal tumor
http://purl.obolibrary.org/obo/MONDO_0016744 | primary melanocytic tumor of central nervous system
http://purl.obolibrary.org/obo/MONDO_0020524 | primary parathyroid hyperplasia
http://purl.obolibrary.org/obo/MONDO_0015682 | primary peritoneal tumor
http://purl.obolibrary.org/obo/MONDO_0019044 | tumor of hematopoietic and lymphoid tissues
http://purl.obolibrary.org/obo/MONDO_0018191 | tumor of testis and paratestis

**Comment by nicolevasilevsky** [comment]
closed accidentally with the PR

**Comment by nicolevasilevsky** [comment]
note- only the orphaned terms were reviewed. Still to do: review the 'leaves the branch' terms

**Comment by nicolevasilevsky** [comment]
leaves the branch is ready for review

**Comment by sabrinatoro** [comment]
@nicolevasilevsky I have reviewed the "leaves the branch", and I have only 1 comment. Otherwise, it is ready for PR. 
Thank you!

**Comment by nicolevasilevsky** [comment]
updated the spreadsheet per your comment, this is ready for PR

**Comment by sabrinatoro** [comment]
Expert review feedback: https://docs.google.com/spreadsheets/d/1baEDi3WVXvkgxleS3Gwee4xlur4iFCa8/edit#gid=2093035201

TO DO:
- [ ] add new parents ("ROBOT Reviewer-Curator Agree - Stay" Tab)
- [ ] add new parents ("ROBOT Reviewer-Stay Curator-Leave" Tab)


Note: Once the to-do list is done, 9 terms for which reviewers disagreed will remain to be assessed

**Comment by sabrinatoro** [comment]
There are some comments from the community that should be reviewed, but this is future work. This issue can be closed for now.
